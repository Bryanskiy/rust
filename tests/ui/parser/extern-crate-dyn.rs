extern dyn crate foo;
//~^ ERROR can't find crate for `foo`
//~| `extern dyn` annotation is only allowed with `v0` mangling scheme

fn main() {}
