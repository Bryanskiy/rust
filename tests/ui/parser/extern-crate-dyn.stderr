error[E0463]: can't find crate for `foo`
  --> $DIR/extern-crate-dyn.rs:1:1
   |
LL | extern dyn crate foo;
   | ^^^^^^^^^^^^^^^^^^^^^ can't find crate

error: `extern dyn` annotation is only allowed with `v0` mangling scheme
  --> $DIR/extern-crate-dyn.rs:1:1
   |
LL | extern dyn crate foo;
   | ^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0463`.
